config/cluster-gizmo.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Comment on inclusion of `config/cluster-gizmo.json`. It's best to keep PRs and feature branches as clean as possible. Inclusion of a shared slurm config could be a useful feature to include in its own separate PR, but for ease of review please remove from this PR.